includes/why-grpc.md

```diff
@@ -0,0 +1,12 @@
+Our example is a simple route mapping application that lets clients get
```

Comment:
Add the section heading too?

Reply:
We could do it that way but I’d prefer to keep the heading in the original Markdown files, as it preserves the basic header/text/header/text flow of the original documents

Reply:
I feel like this file is less "why-grpc" and more "basics-intro"